aws_lambda_powertools/logging/logger.py

```diff
@@ -445,19 +439,6 @@ def decorate(event, context, *args, **kwargs):
 
         return decorate
 
-    def setLevel(self, level: Union[str, int]):
```

Comment:
So we dropped this method from the API?

Reply:
no, it's proxied to ._logger via descriptors

Reply:
This isn't in production yet, so it's okay to remove it.

This PR supersedes this other one [PR](https://github.com/awslabs/aws-lambda-powertools-python/pull/2320)